core/templates/dev/head/profile/profile.html

```diff
@@ -128,11 +128,10 @@ <h2 class="oppia-profile-username-small-screen">
       </md-content>
 
       <span class="oppia-profile-portfolio-pages" ng-if="numUserPortfolioExplorations > 6">
-        <li class="glyphicon glyphicon-arrow-left" ng-disabled="currentPageNumber == 0" ng-click="goToPreviousPage()"></li>
-        Showing <[currentPageNumber * PAGE_SIZE + 1]> - <[Math.min(numUserPortfolioExplorations, (currentPageNumber + 1) * PAGE_SIZE)]> of <[numUserPortfolioExplorations]>
-        <li class="glyphicon glyphicon-arrow-right" ng-disabled="(currentPageNumber + 1) * PAGE_SIZE >= userEditedExplorations.length"
-            ng-click="goToNextPage()">
-        </li>
+        <i class="material-icons md-18" ng-disabled="currentPageNumber == 0"
+           ng-click="goToPreviousPage()">&#xE5C4;</i>
+          Showing <[currentPageNumber * PAGE_SIZE + 1]> - <[Math.min(numUserPortfolioExplorations, (currentPageNumber + 1) * PAGE_SIZE)]> of <[numUserPortfolioExplorations]>
```

Comment:
Nit: indentation here is wrong, it is at the same level as the i-tags so it should be deindented by 2 spaces.